lib/classes/class-settings.php

```diff
@@ -26,7 +26,7 @@
           'on_fly'                 => array('WP_STATELESS_MEDIA_ON_FLY', 'false'), 
           'bucket'                 => array('WP_STATELESS_MEDIA_BUCKET', ''), 
           'root_dir'               => array('WP_STATELESS_MEDIA_ROOT_DIR', ''), 
-          'key_json'               => array('WP_STATELESS_MEDIA_JSON_KEY', ''), 
```

Comment:
If users want to simply put JSON key to the config file as constant.
**key_json** contains json not json file path. That's why we can't use WP_STATELESS_MEDIA_KEY_FILE_PATH here.